pkg/commands/process/settings/rules/javascript/express/external_file_upload.yml

```diff
@@ -0,0 +1,21 @@
+patterns:
+  - pattern: res.sendFile($<EXPRESS_REQ>)
```

Comment:
This also matches on something like the following (additional things following `$<EXPRESS_REQ>`, which is unexpected to me since there is no `$<...>`. 

Any tips on how to exclude this case, if it's possible?

```javascript
res.sendFile(req.params.file, { root: path.join(__dirname, 'public') })
```

Reply:
not sure if that's the expected behaviour either

Reply:
I would expect the pattern to only match a single argument. Let's have a look at this one together